Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 # Miniconda - Python 3.6, 64-bit, x86, latest
 ARG CONDA_ADD_PACKAGES=""
 ARG PIP_ADD_PACKAGES=""
-ARG TENSORFLOW_VERSION="2.0.0-b1"
+ARG TENSORFLOW_VERSION="2.0.0a0"
```

Comment:
Will find out why keras model predict will hang at the end using `tensorflow==2.0.0b1`

Reply:
For the record, maybe file an issue.

Reply:
Added in https://github.com/sql-machine-learning/sqlflow/issues/573